### DISCUSSION TITLE
I  don't know how to fix it

### DISCUSSION THREAD

**Discussion by DaffaAgastya271** [Monika-After-Story/MonikaModDev#8499] [QA - tech support + general] [answered]
can someone help me please
![IMG_20211220_231604](https://user-images.githubusercontent.com/96450947/146846478-6b9408e3-3cc1-472a-a628-490b31e8da99.jpg)

**Comment by multimokia** [comment, marked as answer]
Check out [this link](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#all-the-buttons-look-like-this) on our FAQ
